compiler/modules/front/expr.bal

```diff
@@ -510,7 +510,13 @@ function codeGenArithmeticBinaryExpr(ExprContext cx, bir:BasicBlock bb, bir:Arit
             return { result: { value, semType: resultType }, block: bb };
         }
         result = cx.createTmpRegister(t:INT, pos);
-        bir:IntArithmeticBinaryInsn insn = { op, pos, operands, result };
+        bir:Insn insn;
+        if isNonPanickingIntOp(operands[0], operands[1], op) {
```

Comment:
I wrote a comment but it disappeared. Write like this

```
var name = intArithmeticOpNeverPanics(op, operands);
insn = { op, pos, name, operand, result };
```

Reply:
Fixed. Not sure if this is a bug but if I use name directly as `insn = { op, pos, name, operand, result };` (without if) compiler complains saying `a type compatible with mapping constructor expressions not found in type 'wso2/nballerina.bir:0.1.0:Insn'`

Reply:
Yes, I think that is a JBUG: I seem to remember coming across something similar.